### PR TITLE
perf: yarn patch dayjs

### PR DESCRIPTION
**Pull request by keithwillcode**
## What does this PR do?

Fixes #13358 

Extension of #13512 but used yarn patch to put the changes directly into the dayjs package

## Type of change

- Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- Ensure test suite is passing and date functionality across the system is stable

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,7 +116,10 @@
     "@types/react-dom": "^18.0.9",
     "next-i18next@^13.2.2": "patch:next-i18next@npm%3A13.3.0#./.yarn/patches/next-i18next-npm-13.3.0-bf25b0943c.patch",
     "libphonenumber-js@^1.10.51": "patch:libphonenumber-js@npm%3A1.10.51#./.yarn/patches/libphonenumber-js-npm-1.10.51-4ff79b15f8.patch",
-    "libphonenumber-js@^1.10.12": "patch:libphonenumber-js@npm%3A1.10.51#./.yarn/patches/libphonenumber-js-npm-1.10.51-4ff79b15f8.patch"
+    "libphonenumber-js@^1.10.12": "patch:libphonenumber-js@npm%3A1.10.51#./.yarn/patches/libphonenumber-js-npm-1.10.51-4ff79b15f8.patch",
+    "dayjs@1.11.2": "patch:dayjs@npm%3A1.11.4#./.yarn/patches/dayjs-npm-1.11.4-97921cd375.patch",
```
Comment (keithwillcode) [inline package.json:120]: Not sure why this patch for 1.11.2 is taking the 1.11.4 patch but everything is staying stable from the test suite.

File: package.json [hunk 2]
```diff
@@ -116,7 +116,11 @@
     "@types/react-dom": "^18.0.9",
     "next-i18next@^13.2.2": "patch:next-i18next@npm%3A13.3.0#./.yarn/patches/next-i18next-npm-13.3.0-bf25b0943c.patch",
     "libphonenumber-js@^1.10.51": "patch:libphonenumber-js@npm%3A1.10.51#./.yarn/patches/libphonenumber-js-npm-1.10.51-4ff79b15f8.patch",
-    "libphonenumber-js@^1.10.12": "patch:libphonenumber-js@npm%3A1.10.51#./.yarn/patches/libphonenumber-js-npm-1.10.51-4ff79b15f8.patch"
+    "libphonenumber-js@^1.10.12": "patch:libphonenumber-js@npm%3A1.10.51#./.yarn/patches/libphonenumber-js-npm-1.10.51-4ff79b15f8.patch",
+    "dayjs": "1.11.2",
```
Comment (zomars) [inline package.json:120]: @keithwillcode added a single version resolution to we only need to patch once. Also remade an re-applied the patch.

**Review by zomars — DISMISSED** [review]
LFG!

File: package.json (hunk 2 above)
Comment (keithwillcode) [reply package.json:120]: Good call. Thank you 🙏

**Review by zomars — APPROVED** [review]

**Pull request merged**